ktlint-ruleset-experimental/src/test/resources/spec/indent/format-property-accessor-expected.kt.spec

```diff
@@ -1,9 +1,8 @@
 class C {
 
     private val Any.className
-        get() =
-            this.javaClass.name
-                .fn()
+        get() = this.javaClass.name
```

Comment:
am also unsure here.

Reply:
both look bad to me, but the extra newline irks me. maybe it's just me. :)